### PR TITLE
fix(results): Fix results ordering and visibility

### PR DESCRIPTION
**Pull request by soyacz**
In case we have multiple tables, their order is random.
Added ordering based on which table values were inserted first.

Results were having to little margins and looking too messy.
Added filtering capabilities, increased margins and shown a vertical bar
on the left side of each table with color based on results (red if any
result was failed).

### CODE REVIEW

**Comment by soyacz** [comment]
image

**Review by fruch — APPROVED** [review]
LGTM

File: argus/backend/service/testrun.py [hunk 1]
```diff
@@ -307,23 +307,27 @@ def get_run_events(self, run_id: UUID):
         }
         return response
 
-    def fetch_results(self, test_id: UUID, run_id: UUID) -> dict:
-        query_fields = ["column", "row", "value", "status"]
+    def fetch_results(self, test_id: UUID, run_id: UUID) -> list[dict]:
+        session = ScyllaCluster.get_session()
```
Comment (k0machi) [inline argus/backend/service/testrun.py:311]: you should probably use `ScyllaCluster.prepare` to get a `PreparedStatement` statement instead

**Review by k0machi — CHANGES_REQUESTED** [review]

File: argus/backend/service/testrun.py (hunk 1 above)
Comment (soyacz) [reply argus/backend/service/testrun.py:311]: true, fixed, please take  a look again

**Comment by soyacz** [comment]
when merged we can deploy it (along with graphs support)

File: argus/backend/service/testrun.py (hunk 1 above)
Comment (k0machi) [reply argus/backend/service/testrun.py:311]: Ah, I meant this [helper method](https://github.com/scylladb/argus/blob/master/argus/backend/db.py#L70-L75) as it also saves this statement so you don't have to on every query.

Comment (soyacz) [reply argus/backend/service/testrun.py]: ok, I see, fixed.

**Review by k0machi — APPROVED** [review]

**Pull request merged**